CondFormats/HcalObjects/interface/HcalQIEType.h

```diff
@@ -12,6 +12,7 @@ contains the QIE Typese + corresponding DetId
 class HcalQIEType
 {
  public:
+  enum {QIE8=0, QIE10=1, QIE11=2};
```

Comment:
Indeed, I thought it was acceptable to add the enum to the CondFormat since it is not persistent.

If anything, we will use the enum more heavily in the future, as the HCAL upgrades progress and more new QIEs are installed. I do not think HcalDbHardcode is the right place to define it, since the enum will be used to interpret *both* database and hardcode conditions.